app/coffee/models/Project.coffee

```diff
@@ -17,22 +17,25 @@ DeletedDocSchema = new Schema
 ProjectSchema = new Schema
 	name              :   {type:String, default:'new project'}
 	lastUpdated       :   {type:Date, default: () -> new Date()}
-	lastOpened		  :   {type:Date}
-	active			  :   { type: Boolean,  default: true }
+	lastOpened        :   {type:Date}
+	active            :   { type: Boolean,  default: true }
 	owner_ref         :   {type:ObjectId, ref:'User'}
 	collaberator_refs :   [ type:ObjectId, ref:'User' ]
 	readOnly_refs     :   [ type:ObjectId, ref:'User' ]
 	rootDoc_id        :   {type: ObjectId}
 	rootFolder        :   [FolderSchema]
 	publicAccesLevel  :   {type: String, default: 'private'}
-	compiler		  :   {type:String, default:'pdflatex'}
+	compiler          :   {type:String, default:'pdflatex'}
 	spellCheckLanguage :   {type:String, default:'en'}
 	deletedByExternalDataSource : {type: Boolean, default: false}
 	description : {type:String, default:''}
 	archived          : { type: Boolean }
 	deletedDocs       : [DeletedDocSchema]
 	imageName         : { type: String }
 	track_changes     : { type: Object }
+	overleaf          :
+		id              : { type: Number }
+		imported_at_version : { type: Number }
```

Comment:
Overleaf actually uses both `version` (for the new history service) and `ver_id` for the main storage system. I think this would be a `ver_id`, correct? Maybe best to actually say `ver_id`.